libs/auth/scopes/src/lib/licenses.scope.ts

```diff
@@ -0,0 +1,10 @@
+export enum LicenseScope {
+  // Client can read all licenses
+  read = '@island.is/licenses:read',
+
+  // Client can generate licenses to display (e.g. pkpass for drivers license)
+  generate = '@island.is/licenses:generate',
```

Comment:
The plan is to start with a shared `@island.is/internal` scope for most APIs until we have a use case for explicit scopes (either delegations or external clients). 

I don't know ... `GenericScope.internal`? :)

Reply:
OK, will add, the generic scope, name sounds fine.